### ISSUE TITLE
Darknet Learning and Learning Issues of Rtx3090

### ISSUE THREAD

**Issue by LanguageFlowCho** [AlexeyAB/darknet#7115]
Hello friends Today I came to discuss the RTX 3090 learning related issues.
When learning Rtx3090, I do not know why, but the map value of learning comes out as 0%.

I checked the train value and the valid value correctly, divided the value, and checked the cfg value, but there is no problem.

This is Yolov3-tiny3l.
[net]
# Testing
#batch=1
#subdivisions=1
# Training
batch=64
subdivisions=2
width=608
height=608
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.00261
burn_in=1000
max_batches = 20000
policy=steps
steps=16000,18000
scales=.1,.1

[convolutional]
batch_normalize=1
filters=32
size=3
stride=2
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=64
size=3
stride=2
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[route]
layers=-1
groups=2
group_id=1

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[route]
layers = -1,-2

[convolutional]
batch_normalize=1
filters=64
size=1
stride=1
pad=1
activation=leaky

[route]
layers = -6,-1

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[route]
layers=-1
groups=2
group_id=1

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[route]
layers = -1,-2

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[route]
layers = -6,-1

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[route]
layers=-1
groups=2
group_id=1

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[route]
layers = -1,-2

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[route]
layers = -6,-1

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

##################################

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=21
activation=linear

[yolo]
mask = 3,4,5
anchors = 393,493, 412,494, 404,519, 396,550, 419,541, 418,584
classes=2
num=6
jitter=.3
scale_x_y = 1.05
cls_normalizer=1.0
iou_normalizer=0.07
iou_loss=ciou
ignore_thresh = .7
truth_thresh = 1
random=1
resize=1.5
nms_kind=greedynms
beta_nms=0.6

[route]
layers = -4

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[upsample]
stride=2

[route]
layers = -1, 23

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=21
activation=linear

[yolo]
mask = 1,2,3
anchors = 393,493, 412,494, 404,519, 396,550, 419,541, 418,584
classes=2
num=6
jitter=.3
scale_x_y = 1.05
cls_normalizer=1.0
iou_normalizer=0.07
iou_loss=ciou
ignore_thresh = .7
truth_thresh = 1
random=1
resize=1.5
nms_kind=greedynms
beta_nms=0.6

![chart_mask_tiny_3l](https://user-images.githubusercontent.com/54011340/102027718-6d93a900-3de9-11eb-93b1-471ed4a73afc.png)

I need your help. If you have experience or knowledge, discuss it.

**Comment by BotCyborg** [comment]
Make sure you are using CUDA 11.1 and cuDNN v8.0.5. Older version of CUDA and cuDNN are known to have problems running darknet on RTX3000 series cards

**Comment by LanguageFlowCho** [comment]
BotCyborg Thank you for your answer.
The contents of the learning by applying the stated setting are shown as the result above.
Did the same thing happen to you? I wonder if there is any solution.

**Comment by bulatnv** [comment]
> I came to discuss the RTX 3090 learning related issues

Hello, @LanguageFlowCho.
There are no issues. Training works fine, there is no difference in training with previous architecture, except for speed and weird loss graph.

**Comment by LanguageFlowCho** [comment]
bulatnv Thanks for the reply.
Thanks to this, the problem has been resolved, and the issue is closed.
